File: ThingiBrowse/Classes/ThingResultListItem.cs
using System;
using System.Net;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Documents;
using System.Windows.Ink;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Animation;
using System.Windows.Shapes;

namespace ThingiBrowse
{
    public class ThingResultListItem
    {
        public String ThingTitle { get; set; }
        public String ThingCreatedBy { get; set; }
        public String ThingTime { get; set; }
        public String ThingUrl { get; set; }
        public String ThingCreatorUrl { get; set; }
        public String ThingImageUrl { get; set; }

        public ThingResultListItem(String ThingTitle, String ThingCreatedBy, String ThingTime, String ThingUrl, String ThingCreatorUrl, String ThingImageUrl)
        {
            this.ThingTitle = ThingTitle;
            this.ThingCreatedBy = ThingCreatedBy;
            this.ThingTime = ThingTime;
            this.ThingUrl = ThingUrl;
            this.ThingCreatorUrl = ThingCreatorUrl;
            this.ThingImageUrl = ThingImageUrl;
        }

    }
}

File: Android/src/com/blogspot/marioboehmer/thingibrowse/domain/ThingResultListItem.java
/*   Copyright 2012 Mario Böhmer
 *
 *   Licensed under Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0) 
 *   you may not use this file except in compliance with the License.
 *   You may obtain a copy of the License at
 *
 *       http://creativecommons.org/licenses/by-nc-sa/3.0/
 */
package com.blogspot.marioboehmer.thingibrowse.domain;

import java.io.Serializable;

import android.widget.ListView;

import com.blogspot.marioboehmer.thingibrowse.fragments.ThingResultListFragment;

/**
 * Domain object to represent an item in the {@link ThingResultListFragment}'s
 * {@link ListView}.
 * 
 * @author Mario Böhmer
 */
public class ThingResultListItem implements Serializable {

	private static final long serialVersionUID = -3964683443697700389L;

	private String thingTitle;
	private String thingCreatedBy;
	private String thingTime;
	private String thingUrl;
	private String thingCreatorUrl;
	private String thingImageUrl;

	public ThingResultListItem(String thingTitle, String thingCreatedBy,
			String thingTime, String thingUrl, String thingCreatorUrl,
			String thingImageUrl) {
		this.thingTitle = thingTitle;
		this.thingCreatedBy = thingCreatedBy;
		this.thingTime = thingTime;
		this.thingUrl = thingUrl;
		this.thingCreatorUrl = thingCreatorUrl;
		this.thingImageUrl = thingImageUrl;
	}

	public String getThingTitle() {
		return thingTitle;
	}

	public String getThingCreatedBy() {
		return thingCreatedBy;
	}

	public String getThingTime() {
		return thingTime;
	}

	public String getThingUrl() {
		return thingUrl;
	}

	public String getThingCreatorUrl() {
		return thingCreatorUrl;
	}

	public String getThingImageUrl() {
		return thingImageUrl;
	}

}
